_data/simple-icons.json

```
@@ -8786,6 +8786,11 @@
             "source": "https://www.liquibase.com/brand",
             "guidelines": "https://www.liquibase.com/brand"
         },
+        {
+            "title": "Listmonk",
```

The correct capitalization of the project is "listmonk".
```suggestion
            "title": "listmonk",
```